docs/guides/flux.md

```diff
@@ -110,35 +110,74 @@ So we now have an action called `userValueUpdated`, which when called dispatches
 ### Connecting a React Component to our Flux Store
 
 The store is now updating its data - but we have no React components connected to the store! Lets set one up:
+Here is an example of how to set up a connected Component. It shows several ways that a component can connect to Stores or simply be overridden with props.
 
 ```js
 // ./src/views/user/index.js
 
 import React from 'react';
 import { connect } from 'utils/flux';
-import Textbox from 'carbon-react/lib/components/textbox';
+import AccountInvitation from 'test/lib/components/accountinvitation';
 import UserStore from 'stores/user';
+import GiftsStore from 'stores/gifts';
 import UserActions from 'actions/user';
 
-class UserView extends React.Component {
+class AccountInvitation extends React.Component {
   render() {
     return (
-      <Textbox name="foobar"
-        value={ this.state.userStore.get('foobar') }
-        onChange={ UserActions.userValueUpdated } />
+      <div>
+        <Textbox name="invitationid"
+          value={ props.invitationid }
+        />
+        <Textbox name="firstName"
+          value={ props.firstname }
+          onChange={ UserActions.userFirstNameValueUpdated } />
+        />
+        <Textbox name="surname"
+          value={ props.surname }
+          onChange={ UserActions.useSurnameValueUpdated } />
+        />
+        <Textbox name="gift"
+          value={ props.gift }
+        />
+      </div>
     );
   }
 }
+const mapStateToProps = (userState, giftsState, props) => {
+  return {
+    invitationid: props.invitationid,
+    firstname: props.firstname || userState.get('firstname'),
+    surname: props.surname ||userState.get('surname'),
+    gift: giftsState.get('gift')
```

Comment:
You've got a typo in there `props.surname ||userState.get('surname')`, you're missing a space. I actually think it was OK when you just had the `props.invitationid` and no other mention of props in there.